CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 1.2.1-dev
 
+* Added `serveGuarded` and `serveRequestGuarded` API that always serve
```

Comment:
Should make this 1.3.0(-dev) – since we're adding API

Reply:
sure:) I wanted to get the API right first...